src/components/PopupWithConfirmation.js

```diff
@@ -8,8 +8,8 @@ export default class PopupWithConfirmation extends Popup {
     this._submitButtonText = this._submitButton.textContent;
   }
 
-  open() {
-    super.open();
+ {
+    super();
   }
```

Comment:
```js

 {
    super();
  }
```

* The app starts with errors https://disk.yandex.ru/i/tj8g5mr0Ih0_ig 


Please, delete this `super`